app/lib/rocketchat/services/shareExtension.ts

```diff
@@ -0,0 +1,92 @@
+import { Q } from '@nozbe/watermelondb';
+
+import { shareSetSettings, shareSelectServer, shareSetUser } from '../../../actions/share';
+import SSLPinning from '../../../utils/sslPinning';
+import log from '../../../utils/log';
+import { IShareServer, IShareUser } from '../../../reducers/share';
+import UserPreferences from '../../userPreferences';
+import database from '../../database';
+import RocketChat from '..';
+import { encryptionInit } from '../../../actions/encryption';
+import { store } from '../../auxStore';
+import sdk from './sdk';
+
+export async function shareExtensionInit(server: string) {
```

Comment:
- Open Photos on iOS sim
- Try to share a photo to our app
- Share extension opens, but Reactotron raises this error
<img width="724" alt="image" src="https://user-images.githubusercontent.com/804994/154553842-173d7fdf-94e8-45da-a49e-9e02407f2bcf.png">